bower.json

```diff
@@ -1,19 +1,24 @@
 {
-  "name": "vui-table",
+  "name": "d2l-table",
   "main": "table.scss",
   "license": "Apache-2.0",
   "ignore": [
     "node_modules",
     "bower_components",
```

Comment:
Technically, no need to ignore either of these since they're not checked in anyway.